packages/lib-react-components/src/Markdownz/helpers/storybookExamples.js

```diff
@@ -0,0 +1,152 @@
+import dedent from 'dedent'
+
+export const examples = [
+  {
+    label: 'emoji',
+    content: ':smile: :-)'
+  },
+  {
+    label: 'bold',
+    content: '**bold**'
+  },
+  {
+    label: 'italicized',
+    content: '_italicized_'
+  },
+  {
+    label: 'anchor',
+    content: '[link](https://www.zooniverse.org/)'
+  },
+  {
+    label: 'relative link',
+    content: '[relative-link](/relative)'
+  },
+  {
+    label: 'image',
+    content: '![imagealttext](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)'
+  },
+  {
+    label: 'image resized like TESS',
+    content: '![like TESS](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg =100x)'
+  },
+  {
+    label: 'image resized in alt text',
+    content: '![imagealttext =100x100](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg)'
+  },
+  {
+    label: 'image resized in url',
+    content: '![imagealttext](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg =100x100)'
+  },
+  {
+    label: 'image with equals sign in the alt text',
+    content: '![A blackboard showing the expression 2x2=4](https://panoptes-uploads.zooniverse.org/production/subject_location/66094a64-8823-4314-8ef4-1ee228e49470.jpeg =100x100)'
+  },
+  {
+    label: 'video (using image syntax)',
+    content: '![alt](https://static.zooniverse.org/www.zooniverse.org/assets/home-video.mp4)'
+  },
+  {
+    label: 'audio (using image syntax)',
+    content: '![alt](https://panoptes-uploads.zooniverse.org/production/subject_location/1c93591f-5d7e-4129-a6da-a65419b88048.mpga)'
+  },
+  {
+    label: 'JSON data subject',
+    content: '![alt](https://panoptes-uploads.zooniverse.org/subject_location/74fddc9b-790d-47c6-9eac-110c64022ea8.json =300x)'
+  },
+  {
+    label: 'superscript',
+    content: 'super^script^'
+  },
+  {
+    label: 'subscript',
+    content: 'sub~script~'
+  },
+  {
+    label: 'typed email address',
+    content: 'contact@zooniverse.org'
+  },
+  {
+    label: 'Zooniverse user mention',
+    content: '@srallen @team'
+  },
+  {
+    label: 'Zooniverse Talk Tag mention',
+    content: '#tiger'
+  },
+  {
+    label: 'Zooniverse Subject mention (only works in project context)',
+    content: '^S1234'
+  },
+  {
+    label: 'Link inside a link',
+    content: '[foo @username bar](http://example.com)'
+  },
```

Comment:
This rendered example does not match the deployed version. The markdown string is the same though, so please run the storybook locally and take a look at what might be happening:

This branch:
<img width="294" alt="Screenshot 2023-10-24 at 12 34 19 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/5c147ac6-8c4d-451d-8b41-0ea95db8a481">


Deployed [version](https://zooniverse.github.io/front-end-monorepo/@zooniverse/react-components/index.html?path=/story/components-markdownz--default):
<img width="174" alt="Screenshot 2023-10-24 at 12 34 26 PM" src="https://github.com/zooniverse/front-end-monorepo/assets/23665803/419b781c-1a12-4bc2-8db7-abadcb0a9b02">

Reply:
This is expected for `markdownz`. If you try it in Talk, or in a PFE project page, it won't work either. 

I guess the remark version fixed this.

Reply:
The `markdownz` code for mentions is a regex that's looking for strings preceded by @. 
https://github.com/zooniverse/markdownz/blob/22dc7ecfbb8424967e8f94ef0b0d6ee9e712d6f0/src/lib/default-transformer.js#L4

Reply:
Sounds good thanks! This isn’t in the PFE markdown guide, so I’m fine with just using `markdownz`'s handling. I honestly can't remember why the link inside a link example was added to FEM's storybook.

Reply:
It is an odd example. I don’t think I’ve seen anyone put an @-mention inside a link either. 

I would guess that hashtags and subject mentions can’t be placed inside links either.